src/System.Management.Automation/engine/ExperimentalFeature/ExperimentalFeature.cs

```diff
@@ -114,7 +114,10 @@ static ExperimentalFeature()
                     description: "New parameter set for ForEach-Object to run script blocks in parallel"),
```

Comment:
@adityapatwardhan, your last commit had 2 failures in `PowerShell-CI-macos`
Format-Custom with expression based EntrySelectedBy in a CustomControl.Property expression binding should be able to access the current object
```powershell
Expected strings to be the same, but they were different.
Expected length: 49
Actual length:   48
Strings differ at index 29.
Expected: '...erty^^Name...'
But was:  '...erty^Name^...'
at <ScriptBlock>, /Users/vsts/agent/2.155.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Custom.Tests.ps1: line 410
410:         $ps.Invoke() -replace '\r?\n', "^" | Should -BeExactly $expectedOutput
```
Format-Custom with expression based EntrySelectedBy in a CustomControl.ScriptBlock expression binding should be able to access the current object
```powershell
Expected strings to be the same, but they were different.
Expected length: 62
Actual length:   61
Strings differ at index 32.
Expected: '...lock^^Name...'
But was:  '...lock^Name^...'
at <ScriptBlock>, /Users/vsts/agent/2.155.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Custom.Tests.ps1: line 437
437:         $ps.Invoke() -replace '\r?\n', "^" | Should -BeExactly $expectedOutput
```